tests/unit/Alteration.spec.ts

```diff
@@ -311,7 +311,7 @@ describe('Alteration component', () => {
     // Validate People And Roles
     expect(store.state.stateModel.peopleAndRoles.orgPeople[0].officer.firstName).toBe('CAMERON')
     expect(store.state.stateModel.peopleAndRoles.orgPeople[0].officer.lastName).toBe('BOWLER')
-    expect(store.state.stateModel.peopleAndRoles.orgPeople[0].roles[0].roleType).toBe('Director')
+    expect(store.state.stateModel.peopleAndRoles.orgPeople[0].role).toBe('director')
```

Comment:
I think a party can have more than 1 role (ie, array)?

Reply:
This is a `/director` response where the property `role` is a singular string value.
Different from our more recently updated `roles` property which is part of our `/parties` response.

Reply:
Alterations only calls `/directors` not the `/parties`
I think `/parties` is actually somewhat new, possibly introduced at the start of firms? I'm not sure 🤔

Reply:
Actually, I think `/directors` is the new endpoint. I only see `/parties` used in Filings UI (not Create UI). So this is probably fine. This doesn't even violate the schema because the directors schema doesn't mention roles 🤦‍♂️

Reply:
It does violate the [party](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/party.json) schema though.

We just have to remember that directors are not a subclass of party.

Reply:
Directors is what we use for Alterations, so its been around for a bit i think. 

But yea, this is role because thats the property returned in the /directors response. I'm not sure if it was updated recently.

Reply:
> It does violate the [party schema](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/party.json) though.
> 
> We just have to remember that directors are not a subclass of party.

For sure